tools/ctl/ctl.c

```diff
@@ -20,6 +20,13 @@
 #define BUFFER_SIZE_OFFSET	1
 #define BUFFER_ABI_OFFSET	2
 
+/* Definitions for multiple IPCs */
+enum sof_ipc_type {
+	SOF_IPC,
```

Comment:
Nitpick: but important.
SOF_IPC_TYPE_ prefix needed for all, IPC4 is generic and NOT Intel only.

Reply:
We need same update in the kernel.

Reply:
Sure, but we first need to agree on the change, hence I was asking.